### ISSUE TITLE
Data queries sections identical in JSON provided in EDR API Reference

### ISSUE THREAD

**Issue by DPeterK** [opengeospatial/ogcapi-environmental-data-retrieval#347]
In the JSON examples for `/collections` and `/collections/{collectionID}` in the [API Reference](https://developer.ogc.org/api/edr/edr_api.html), the example content for all `data_queries` types is identical, and they all appear to contain the content for a `position` data query. 

For example, here's the top of the `data_queries` section, including both `position` and `radius` (which appears to be identical to `position`): 
```JSON

  "data_queries": {
    "position": {
      "link": {
        "href": "http://data.example.com/collections/monitoringsites/locations/1234",
        "rel": "alternate",
        "type": "application/geo+json",
        "hreflang": "en",
        "title": "Monitoring site name",
        "length": 0,
        "templated": true,
        "variables": {
          "title": "Position query",
          "description": "Query to return data for a defined well known text point",
          "query_type": "trajectory",
          "coords": {
            "description": "string",
            "type": "string",
            "example": "string"
          },
          "within_units": [
            "km",
            "miles"
          ],
          "width_units": [
            "km",
            "miles"
          ],
          "height_units": [
            "m",
            "hPa"
          ],
          "output_formats": [
            "CoverageJSON",
            "GeoJSON",
            "IWXXM",
            "GRIB"
          ],
          "default_output_format": "string",
          "crs_details": [
            {
              "crs": "native",
              "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
            }
          ]
        }
      }
    },
    "radius": {
      "link": {
        "href": "http://data.example.com/collections/monitoringsites/locations/1234",
        "rel": "alternate",
        "type": "application/geo+json",
        "hreflang": "en",
        "title": "Monitoring site name",
        "length": 0,
        "templated": true,
        "variables": {
          "title": "Position query",
          "description": "Query to return data for a defined well known text point",
          "query_type": "trajectory",
          "coords": {
            "description": "string",
            "type": "string",
            "example": "string"
          },
          "within_units": [
            "km",
            "miles"
          ],
          "width_units": [
            "km",
            "miles"
          ],
          "height_units": [
            "m",
            "hPa"
          ],
          "output_formats": [
            "CoverageJSON",
            "GeoJSON",
            "IWXXM",
            "GRIB"
          ],
          "default_output_format": "string",
          "crs_details": [
            {
              "crs": "native",
              "wkt": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.01745329251994328,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]]"
            }
          ]
        }
      }
    }
```
This is taken from the JSON example for `/collections/{collectionID}`, but identical content is provided in the example for `/collections` too.

I take it these aren't intended to be identical?

Incidentally the same duplication isn't present in the [EDR Standard Doc, §D.4](https://docs.ogc.org/is/19-086r4/19-086r4.html#toc68), where each data query has its own distinct set of values.

**Comment by DPeterK** [comment]
cc @m-burgoyne, to whom I mentioned this briefly yesterday.

**Comment by chris-little** [comment]
Fixed by PR #350
